lib/active_record/connection_adapters/sqlserver_adapter.rb

```diff
@@ -418,14 +418,12 @@ def configure_connection ; end
       def configure_application_name ; end
 
       def initialize_dateformatter
-        @database_dateformat = user_options_dateformat
-        a, b, c = @database_dateformat.each_char.to_a
-        [a, b, c].each { |f| f.upcase! if f == 'y' }
-        dateformat = "%#{a}-%#{b}-%#{c}"
+        # ISO 8601 format
+        dateformat = "%Y-%m-%d"
```

Comment:
Prefer descriptive variable names vs comments :) but we really don't need either.